docs/tutorial/rln-chat2-live-testnet.md

```diff
@@ -0,0 +1,125 @@
+# Building a local network of spam-protected chat2 clients 
+
+This document is a tutorial on how to run chat2 in spam-protected/rate-limited mode on a designated content topic  `/toy-chat/2/luzhou/proto` using the waku-RLN-Relay protocol.
+You will connect your chat2 client to waku2 test fleets.
+Note that test fleets will not filter spam messages, they merely route messages.
+Spam detection takes place at the chat2 users.
```

Comment:
You meant at the chat2 users' end?

Reply:
Right, fixed in cff80e82d3e93162b747b519b60921dc355b172b